lua/astrocommunity/pack/typescript/init.lua

```diff
@@ -233,4 +233,13 @@ return {
       },
     },
   },
+  {
+    "nvim-neotest/neotest",
+    optional = true,
+    dependencies = { "nvim-neotest/neotest-jest" },
```

Comment:
```suggestion
    dependencies = { { "nvim-neotest/neotest-jest", config = function() end } },
```

Reply:
https://github.com/AstroNvim/astrocommunity/commits/main/?since=2024-09-04&until=2024-09-05

just pushed a similar fix everywhere else we have neotest adapters that can be configured